scripts/tests/run_test_suite.py

```diff
@@ -295,10 +295,16 @@ def cmd_list(context):
     default=0,
     show_default=True,
     help='Number of tests that are expected to fail in each iteration.  Overall test will pass if the number of failures matches this.  Nonzero values require --keep-going')
+@click.option(
```

Comment:
Should this be conditioned on sys.platform?

Reply:
Added a help message which indicates that this option is available only on Linux, and a runtime check to fail-fast if someone passes that option on other platforms.